applications/OptimizationApplication/tests/test_model_part_utils.py

```diff
@@ -0,0 +1,230 @@
+import KratosMultiphysics as Kratos
```

Comment:
could you please make a separate folder for the test_model_part_utils and move its stuff like mdpas there . Like the other tests.

Reply:
done :)